packages/client/src/documents/operations.graphql

```diff
@@ -93,6 +93,14 @@ query getConversation($id: ID!) {
   }
 }
 
+query getAllAddresses($inputValue: String!) {
```

Comment:
The name of this query is a little misleading. I think something like "getMatchingAddresses" might be more descriptive.